images/kibana/README.md

```diff
@@ -22,4 +22,4 @@ docker run -t -p 5601:5601 -e "ELASTICSEARCH_URL=http://myElasticSearchHost:9200
 
 ## Tags
 
-- 5.3.0-1, 5.3, latest
+- 5.3.0-2, 5.3, latest
```

Comment:
5.3.0-2 is already planned in PR #909 
please rebase once it's merged and update to 5.3.0-3.